perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -138,11 +138,27 @@ sub zurich_public_response_states {
     return wantarray ? keys %{ $states } : $states;
 }
 
+sub zurich_user_response_states {
+    my $states = {
+        'hidden'          => 1,
+        'investigating'   => 1, # wish
+        'partial'         => 1, # not contactable
```

Comment:
The comment a bit above this that is a copy of this is I guess not needed any more.